scbs/MethylVI_batch.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p>Make consistent with other tutorials.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>@canergen what exactly should I change here? I took this block of code from the [Atlas integration tutorial[(<a href="https://docs.scvi-tools.org/en/stable/tutorials/notebooks/scrna/harmonization.html" target="_blank">https://docs.scvi-tools.org/en/stable/tutorials/notebooks/scrna/harmonization.html</a>) </p><p> </p>

Reply:
<p>Thanks and my bad. It was more a note to Ori to double-check once. Resolved.</p>